src/__tests__/GoMath_positionId.test.ts

```diff
@@ -0,0 +1,181 @@
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 1, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+        ],
+        id: ":bc",
+    },
+    {
+        height: 7,
+        width: 7,
+        board: [
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 1, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+        ],
+        id: ":bc",
+    },
+    {
+        height: 7,
+        width: 7,
+        board: [
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 1, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+        ],
+        id: ":bc",
+    },
+    {
+        height: 7,
+        width: 7,
+        board: [
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 1, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+        ],
+        id: ":bc",
+    },
+    {
+        height: 7,
+        width: 7,
+        board: [
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 1, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+        ],
+        id: ":bc",
+    },
+    {
+        // Test colour independence
+        height: 7,
+        width: 7,
+        board: [
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 2, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0],
+        ],
+        id: ":bc",
+    },
+    {
+        // Test a board with more stones
+        height: 7,
+        width: 7,
+        board: [
+            [0, 0, 0, 0, 0, 1, 0],
+            [0, 0, 0, 0, 2, 0, 0],
+            [0, 2, 0, 0, 0, 0, 0],
+            [1, 0, 0, 0, 0, 0, 0],
+            [0, 0, 2, 0, 0, 0, 0],
+            [0, 0, 0, 1, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 2],
+        ],
+        id: "abdgfd:bccfeega",
+    },
+    {
+        // Same board, opposite colours
+        height: 7,
+        width: 7,
+        board: [
+            [0, 0, 0, 0, 0, 2, 0],
+            [0, 0, 0, 0, 1, 0, 0],
+            [0, 1, 0, 0, 0, 0, 0],
+            [2, 0, 0, 0, 0, 0, 0],
+            [0, 0, 1, 0, 0, 0, 0],
+            [0, 0, 0, 2, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 1],
+        ],
+        id: "abdgfd:bccfeega",
+    },
+] as Array<Testcase>;
+
+test("Position IDs", () => {
+    TEST_BOARDS.forEach((testcase: Testcase) => {
```

Comment:
Nit: You might be interested in [`test.each`](https://jestjs.io/docs/api#testeachtablename-fn-timeout)

Basically made for this use case, but might give better error messages on test failure.